app/views/pages/_terms_text.html.erb

```diff
@@ -5,7 +5,11 @@
     <p>
       By accessing this web site, you are agreeing to be bound by these
       web site Terms and Conditions of Use, our
-      <a href="/privacy">Privacy Policy</a>, all applicable laws and regulations,
+      <% if FeatureFlag.enabled?(:creator_onboarding) && User.with_role(:creator).any? && request.env["PATH_INFO"] == new_admin_creator_setting_path %>
+        Privacy Policy, all applicable laws and regulations,
+      <% else %>
+        <a href="<%= app_url("/privacy") %>">Privacy Policy</a>, all applicable laws and regulations,
```

Comment:
I updated this hard-coded privacy link to use the `app_url` path helper instead to keep the link within this page consistent with the links in other pages, such as the COC above.